nuxeo-i18n-behavior.html

```diff
@@ -54,7 +54,7 @@
       },
 
       refreshI18n() {
-        this.i18n = null;
+        this.i18n = () => {};
```

Comment:
Tested with other solutions as well, including:

```
this.i18n = window.nuxeo.I18n.translate;
this.notifyPath('i18n');
```

But it doesn't seem to work. We get the same behavior as without the previous `this.i18n = null`.